workflow/hosts/awspw.yaml

```diff
@@ -27,5 +27,5 @@ MAKE_ACFTBUFR: 'NO'
 DO_TRACKER: 'NO'
 DO_GENESIS: 'NO'
 DO_METP: 'NO'
-SUPPORT_WAVES: 'NO'
-SUPPORTED_RESOLUTIONS: ['C48', 'C96'] # TODO: Test and support all cubed-sphere resolutions.
+SUPPORT_WAVES: 'YES'
```

Comment:
Since `WW3` will now be supported on all platforms, I think we can remove `SUPPORT_WAVES` from the host files and `setup_expt.py`.

```suggestion
```